web/thesauruses/java/strings.json

```diff
@@ -138,24 +138,24 @@
     },
     "is_all_alphabetical": {
       "name": "Is string all alphabetical characters?",
-      "code": "String str = \"abcdABCD\";\nboolean result = str.matches(\"[a-zA-Z]+\");",
-      "comment": "Returns a boolean value of true if the String contains only alphabets, else it returns false." 
+      "code": "StringUtils.isAlpha(stringToCheck)",
+      "comment": "Uses the Apache Common Lang library" 
```

Comment:
Since these aren't built into Java and only work if you import from a third party library, they should be `not-implemented` instead. 

(Same for the others below)

Reply:
That makes sense - I've updated accordingly. Thank you! :)